drive/utils.bal

```diff
@@ -662,3 +665,20 @@ isolated function uploadFileUsingByteArray(http:Client httpClient, byte[] byteAr
     }
 }
 
+# Utility function to retrieve File content as a binary content with its MIME type.
+# 
+# + httpClient - Drive client
+# + path - GET URI path
+# + return - `FileContent` if successful, `error` if not.
+isolated function generateRecordFileContent(http:Client httpClient, string path) returns @tainted FileContent|error {
+    http:Response response = <http:Response> check httpClient->get(<@untainted>path);
+    if (response.statusCode == http:STATUS_OK) {
+        return {
+            content: check response.getBinaryPayload(),
+            mimeType: response.getContentType()
+        }; 
+    } else {
+        fail error(ERR_EXTRACTING_BINARY_PAYLOAD);
```

Comment:
Shall we extract error detail from the response. Otherwise we loose information.

Reply:
Did the change
Now it's giving the original error as follows
```
                    {
                        "error": {
                        "errors": [
                        {
                        "domain": "global",
                        "reason": "notFound",
                        "message": "File not found: fileId.",
                        "locationType": "parameter",
                        "location": "fileId"
                        }
                        ],
                        "code": 404,
                        "message": "File not found: fileId."
                        }
                        }
                ```